action.yml

```diff
@@ -20,10 +20,23 @@ inputs:
   private_api_key:
     description: ''
     required: true
+  working_dir: # used for s2i-core and s2i-base images only
+    description: ''
+    required: false
 
 runs:
   using: 'composite'
   steps:
+    - name: Set working dir
+      id: working_dir
+      shell: bash
+      run: |
+        if ${{ inputs.working_dir  == "" }}; then
```

Comment:
What about to check if variable is empty like in bash?. But there is one more space in command. Just nitpick.